src/Utility/Function.h

```diff
@@ -114,6 +115,8 @@ namespace utl
     virtual std::vector<Real> evalGradient(const Vec3&) const { return {}; }
     //! \brief Returns the time derivatives of the function as a 1D array.
     virtual std::vector<Real> evalTimeDerivative(const Vec3&) const { return {}; }
+    //! \brief Returns the hessian of the function as a 1D array.
+    virtual std::vector<Real> evalHessian(const Vec3&) const { return {}; }
```

Comment:
Put the `evalHessian` declaration between `evalGradient` and `evalTimeDerivate`, like in ExprFunctions.h

Reply:
ok.